BatteryConditionValidation.c

```diff
@@ -10,12 +10,14 @@
 
 static battCondn_t battCondn_s;
 static battCondn_t prevBattCondn_s;
+static BMS_t bms_s;
```

Comment:
Any way to avoid making this global? For thread-safety and avoiding side-effects